### ISSUE TITLE
12-textAlign.js crashes Sketch

### ISSUE THREAD

**Issue by ff6347** [automat/sketch-context2d#4]
/Users/me/Desktop/sketch-context2d/index.js:61
            throw new Error(err || stderr);
            ^

Error: 2016-08-02 08:41:58.880 coscript[38635:2241888] Exception: (null)

```
at /Users/me/Desktop/sketch-context2d/index.js:61:19
at ChildProcess.exithandler (child_process.js:193:7)
at emitTwo (events.js:106:13)
at ChildProcess.emit (events.js:191:7)
at maybeClose (internal/child_process.js:852:16)
at Socket.<anonymous> (internal/child_process.js:323:11)
at emitOne (events.js:96:13)
at Socket.emit (events.js:188:7)
at Pipe._handle.close [as _onclose] (net.js:492:12)
```

**Comment by automat** [comment]
Caused by `Sketch 3.9` api change.

**Comment by automat** [comment]
Fixed with f76dbc4 , updated text layer creation to `Sketch 3.9`.
